widgetssdk/src/main/java/com/glia/widgets/filepreview/ui/ImagePreviewActivity.kt

```diff
@@ -132,8 +126,8 @@ internal class FilePreviewActivity : AppCompatActivity(), FilePreviewContract.Vi
     }
 
     override fun onDestroy() {
-        Logger.i(TAG, "Destroy File Preview screen")
-        filePreviewController?.onDestroy()
+        Logger.i(TAG, "Destroy Image Preview screen")
```

Comment:
The same here - it's better to leave the existing wording because it's aligned with iOS.